h2o-py/tests/testdir_algos/admissibleml/pyunit_PUBDEV_8075_core_infogram_iris_x_att.py

```diff
@@ -23,7 +23,7 @@ def test_infogram_iris_x_attributes():
     infogram_model.train(x=x, y=target, training_frame=fr)
     
     glm_model1 = H2OGeneralizedLinearEstimator(family = 'multinomial')
-    glm_model1.train(x=infogram_model.extract_x_from_model(), y=target, training_frame=fr)
+    glm_model1.train(x=infogram_model._extract_x_from_model(), y=target, training_frame=fr)
```

Comment:
```suggestion
    glm_model1.train(x=infogram_model, y=target, training_frame=fr)
```
that's the goal of your support of models as `x` values after all

Reply:
I know.  I caught that from looking the test failures.  Also, need to rebase with master since I pushed the modelselection PR.